pkg/server/commands/connectedobjects/connected_objects.go

```diff
@@ -14,20 +15,30 @@ import (
 	"github.com/openfga/openfga/pkg/tuple"
 	"github.com/openfga/openfga/pkg/typesystem"
 	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/trace"
 	"golang.org/x/sync/errgroup"
 )
 
+var tracer = otel.Tracer("openfga/pkg/server/commands/connected_objects")
+
+const (
+	// same values as run.DefaultConfig() (TODO break the import cycle, remove these hardcoded values and import those constants here)
+	defaultResolveNodeLimit        = 25
+	defaultResolveNodeBreadthLimit = 100
+	defaultMaxResults              = 1000
+)
+
 type ConnectedObjectsRequest struct {
 	StoreID          string
 	ObjectType       string
 	Relation         string
-	User             isUserRef
+	User             IsUserRef
 	ContextualTuples []*openfgapb.TupleKey
 }
 
-type isUserRef interface {
+type IsUserRef interface {
```

Comment:
Would you please add an issue to represent this more natively in our protobufs in `openfga/api`? The only reason it is here is because we don't have a good way to distinguish between different user/subject types in a strongly typed/structured way and so this was a first attempt to do so.

Ideally though we'd have a way, in our protobuf API definition, to distinguish between `user:jon`, `group:eng#member`, and `user:*` user/subject types.

Reply:
https://github.com/openfga/api/issues/87